python/examples/example_large_models/example_performance_optimization.ipynb

```diff
@@ -226,6 +226,16 @@
     "        print(f\"Import time increased by at least ~{-percent_change:.0f}%.\")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "6b938781",
+   "metadata": {},
+   "source": [
+    "#### (No) simplification of model expressions\n",
+    "\n",
+    "By default, AMICI will try to perform some basic simplification of model expressions. For complex model expressions, or for large models, this can become costly. It very much depends on the model expressions, whether the benefits outweigh the cost."
```

Comment:
maybe provide a little bit of info on how to enable/disable simplification?

Reply:
jupyter didn't save my notebook :disappointed: